must_triage/inspectors/ocp.py

```diff
@@ -78,3 +78,24 @@ def pod_ready(obj):
                 f"Container '{cs['name']}' in "
                 f"pod '{obj['metadata']['name']}' is not ready")
         return result
+    @staticmethod
```

Comment:
The linter will complain about not having a newline after the previous method declaration